### ISSUE TITLE
isRotation is fragile

### ISSUE THREAD

**Issue by jw3126** [nrc-cnrc/EGSnrc#522]
When I use transformations, I often get warnings like
```
getTransformation: the rotation specified by
   1 0 0
   0 0.707107 0.707107
   0 -0.707107 0.707107
 is not a rotation
```
In the code it is checked, that `A*A'` is exactly the identity matrix, see [here](https://github.com/nrc-cnrc/EGSnrc/blob/master/HEN_HOUSE/egs%2B%2B/egs_transformations.h#L220).

I think it would be more appropriate to check `norm(A*A' - I) < epsilon`. I can provide a PR if you like to change this.

**Comment by rtownson** [comment]
Yes, I agree that your solution will be more robust. You are welcome to post it as a PR.

**Comment by ftessier** [comment]
Resolved in #525.
